src/js/game/LevelMVC/LevelView.js

```diff
@@ -430,6 +432,13 @@ module.exports = class LevelView {
       "conduit": ["blocks", "Conduit00", -12, 0],
 
       "chest": ["blocks", "Chest0", -12, -20],
+
+      "chestboat": ["blocks", "Chest0", -12, -20],
+      "chestnautilus": ["blocks", "Chest0", -12, -20],
+      "chestprismarine": ["blocks", "Chest0", -12, -20],
+      "chestheartofthesea": ["blocks", "Chest0", -12, -20],
```

Comment:
Rather than having a separate block for each chest type, can we follow the pattern we use for miniblocks?  The code converts `diamondMiniblock` to a diamond miniblock — I'd like to see `diamondChest` convert to a chest that drops a diamond miniblock.

Reply:
Aaah right I'd forgotten about the `diamondMiniblock`. Will that system be fine with the fact that we're just opening it rather than destroying the chest? I had a few different approaches I considered yesterday in terms of getting chests and different levels to dispense different treasure. 🤔 

I'll investigate the diamondMiniblock this morning.